Known-issues.md

```diff
@@ -58,8 +58,7 @@ sudo dbus-daemon --system
 ```
 
 ## systemd/systemctl
-WSL does not supports systemd.
-I recommend use systemctl alternative script or bottle.
+WSL does not supports systemd natively, so I recommend using a systemctl alternative script or bottle for apps that require it.
```

Comment:
Probably it's to use "we" for exact exp...?

Reply:
I fixed it. Thanks for the feedback!